apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx

```diff
@@ -118,69 +118,112 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
-                            {
-                              'font-bold text-success': isDecisionPositive(
-                                isDecisionComponent,
-                                field.value,
-                              ),
-                              'font-bold text-destructive': isDecisionNegative(
-                                isDecisionComponent,
-                                field.value,
-                              ),
-                            },
-                          )}
-                          pattern={pattern}
-                          autoComplete={'off'}
-                          {...field}
-                        />
-                      </FormControl>
-                    )}
-                    <FormMessage />
-                  </FormItem>
-                )}
-              />
-            ),
+          {formData?.map(
+            ({ title, isEditable, type, format, pattern, value, valueAlias, dropdownOptions }) => {
+              return (
+                <FormField
+                  key={keyFactory(valueId, title, `form-field`)}
+                  control={form.control}
+                  name={title}
+                  render={({ field }) => {
+                    if (isDecisionComponent && !value) return null;
+
+                    const isInput = [
+                      !isValidUrl(value) || isEditable,
+                      !isObject(value),
+                      !isArray(value),
+                    ].every(Boolean);
+                    const isSelect = isInput && !!dropdownOptions;
+
+                    return (
+                      <FormItem>
+                        <FormLabel>{toStartCase(camelCaseToSpace(title))}</FormLabel>
+                        {(isObject(value) || isArray(value)) && (
+                          <div
+                            className={`flex items-end justify-start`}
+                            key={keyFactory(valueId, title, `form-field`)}
+                          >
+                            <JsonDialog
+                              dialogButtonText={`View Information`}
+                              json={JSON.stringify(value)}
+                            />
+                          </div>
+                        )}
+                        {isValidUrl(value) && !isEditable && (
+                          <a
+                            key={keyFactory(valueId, title, `form-field`)}
+                            className={buttonVariants({
+                              variant: 'link',
+                              className: '!block cursor-pointer !p-0',
+                            })}
+                            target={'_blank'}
+                            rel={'noopener noreferrer'}
+                            href={value}
+                          >
+                            {(valueAlias as string) ?? value}
```

Comment:
`valueAlias` allows for the value to be a URL and the rendered value to be `View`.